### PR TITLE
Add import from Inspector Issues and improve clarity and grammar on Add events from Inspector

### PR DESCRIPTION
**Pull request by 1212spark**
Putting this up for review to check content, but I still need to add inline images for importing events from Inspector Issues (or can someone please help me with this? I don't know how) 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206526643387956

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…-events-from-inspector**](https://docs-4lvggdsim.avo-dev.app/inspector/add-events-from-inspector) |
|:-:|
|<a href="https://docs-4lvggdsim.avo-dev.app/inspector/add-events-from-inspector">Screenshot of /inspector/add-events-from-inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/MK-US6wFjkL7Xld2f6IRYwPTpxOALDMudRTqnJYpMXPYg6_zBMylSJyvB1h0_MtqKwYzMraIooheyOlufgDyUg.png">(view full size)</a> |







Commit 434836c87c7d9d7422646d9b38e97870f6488e02 (https://docs-4lvggdsim.avo-dev.app).

**Comment by logason** [comment]
One docs page was missing an import for the Callout component, fixed in 45a0426 @1212spark

File: pages/inspector/add-events-from-inspector.mdx [hunk 1]
```diff
@@ -1,45 +1,74 @@
...
+</div>
 
-- Add all sources the event has according to the inspector
-- Add properties that exist in the tracking plan and are on the event in the inspector.
-- Create and add properties that are missing from the tracking plan but are on the event in the inspector
-  - If the property already exists in different casing, it will not be created.
-  - If the property has reported seeing 2 types (int, string for example) it will select the type that Inspector has seen more of.
-  - Each property will be created with presence "Always Sent"
-  - All new properties created will be created as **Event properties**
+##Reviewing your import
```
Comment (logason) [inline pages/inspector/add-events-from-inspector.mdx:33]:
```suggestion
## Reviewing your import
```

File: pages/inspector/add-events-from-inspector.mdx [hunk 2]
```diff
@@ -1,45 +1,74 @@
...
+From here you can:
 
-- Validate that the property does not exist already in the workspace
-  - If a property with different casing already exists, a window will pop up asking you to confirm your creation of the property.
-- If the property has reported seeing 2 types (int, string for example) it will select the type that Inspector has seen more of.
-- Each property will be created with presence "Always Sent"
-- The property will be created as **Event property**
+- Select which events to include in your import
+- Select the sources to include for each event 
+- Select the properties to include for each event
```
Comment (logason) [inline pages/inspector/add-events-from-inspector.mdx:41]:
```suggestion
- Select the properties to include for each event 
- Adjust property types and whether it should be required or optional
```

**Comment by 1212spark** [comment]
@logason suggestions accepted!

**Review by logason — APPROVED** [review]

File: pages/inspector/add-events-from-inspector.mdx [hunk 3]
```diff
@@ -1,45 +1,72 @@
...
 
-## Event not in Tracking Plan
+## Import events and properties in bulk from Inspector Issues view
 
-If the Inspector detects an event that is not in your tracking plan, it will be highlighted with a red tracking plan icon next to the name, along with an "Add" button to add it to your tracking plan. The events view sidebar also has an option to add the event right from there. Click either button to add the event and all it's properties to the tracking plan.
+In addition to providing the "Add to Tracking Plan" button for events not present in the tracking plan within the [Issue Status view](/inspector/inspector-issues-view#event-info),  you can review all of the events and properties missing from the tracking plan.
 
-![Image shows the Inspector events view with "Add" and "Add event to tracking plan" buttons highlighted](/images/inspector/add-event-from-events-view.png)
+If there are events unexpected by the tracking plan detected in Inspector Issues, there will be a "Add X new events" button at the upper right of the main Inspector Issues table.
+![NEED TO INSERT IMAGE HERE, NOT SURE HOW TO UPLOAD IMAGES]
```
[the remark this answers is not in the document]
Comment (bjornj12) [reply pages/inspector/add-events-from-inspector.mdx]: @klaraavo do we want to swap the image that is here ?

[the remark this answers is not in the document]
Comment (bjornj12) [reply pages/inspector/add-events-from-inspector.mdx]: ahh @logason just merged!

**Pull request merged**